astronomy-pipeline/test_astronomy_pipeline.py

```diff
@@ -0,0 +1,271 @@
+"""Test file for the Astronomy API pipeline files."""
+
+# pylint: disable=R0903, R0913, R0917
+
+import base64
+from unittest import mock
+from datetime import date
+
+from psycopg2 import extras
+
+from astronomy_extract_functions import get_db_connection, get_auth_string, get_db_regions
+from astronomy_extract_functions import get_all_body_positions, make_clean_body_dict, get_moon_urls
+from astronomy_extract_functions import refine_bodies_data, get_position_data, fill_region_time_dict
```

Comment:
Could have the imported functions in brackets and have them spread over lines that way rather than repeat the module name

Reply:
Thanks I've never thought of that actually, will do it now.